src/main/java/main/CHScrobbler.java

```diff
@@ -64,6 +64,29 @@ public static void main(String[] args)
             else
                 dataFolder = prop.getProperty(Statics.DATA_FOLDER_PROP_NAME);
 
+            //validate scrobble threshold seconds
+            int scrobbleThresholdSeconds = 25;
```

Comment:
The API rules state that a scrobble will be counted as a scrobble when it has been [30 seconds elapsed](https://www.last.fm/api/scrobbling).

The 4 minute scrobble threshold can remain, but default must be 30 seconds or above.

Reply:
Alright! I added Statics.DEFAULT_SCROBBLE_THRESHOLD and set it to 30.